packages/wallet/src/test-utils/launchNode.ts

```diff
@@ -0,0 +1,177 @@
+import { BaseAssetId } from '@fuel-ts/address/configs';
+import { Provider } from '@fuel-ts/providers';
+import { spawn } from 'child_process';
+import fsSync from 'fs';
+import fs from 'fs/promises';
+import os from 'os';
+import path from 'path';
+import { getPortPromise } from 'portfinder';
+import kill from 'tree-kill';
+
+import type { WalletUnlocked } from '../wallets';
+
+import { defaultChainConfig } from './defaultChainConfig';
+import { generateTestWallet } from './generateTestWallet';
+
+const defaultFuelCoreArgs = ['--vm-backtrace', '--utxo-validation', '--manual_blocks_enabled'];
+
+type LaunchNodeOptions = {
+  chainConfigPath: string;
```

Comment:
Shouldn't this be optional?

```suggestion
  chainConfigPath?: string;
```

Reply:
@arboleya it is already optional for the `launchNodeAndGetWallets` function because `launchNodeOptions` is a `Partial`: 

```ts
launchNodeOptions?: Partial<LaunchNodeOptions>;
```

Reply:
Do you think the logic for going with a default `chainConfig` when one is not provide should live in the `launchNode` function?

Reply:
Same for `consensusKey`

Reply:
Yes, for both. 👍